assets/sass/components/_modal.scss

```diff
@@ -0,0 +1,99 @@
+$close-button-size: $spacingUnit * 2;
+$inputOutlineWidth: 3px;
+$inputOutlineColour: get-color('border', 'outline');
+$inputOutline: $inputOutlineWidth solid $inputOutlineColour;
+
+.modal-dialog {
+    display: none;
+    position: fixed;
+    z-index: 1000;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    overflow-x: hidden;
+    overflow-y: scroll;
+    outline: 0;
+}
+
+.modal-dialog__box {
+    display: block;
+    position: fixed;
+    background: white;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    margin: 0;
+    padding: 0;
+    overflow-y: auto;
+    border: 0;
```

Comment:
Looks like this (eg. fullscreen) on mobile:

![image](https://user-images.githubusercontent.com/394376/69138584-d6a9cf80-0ab6-11ea-80cf-8ba930b1f456.png)